src/types/sidecar-config/CONFIG.ts

```diff
@@ -21,6 +21,7 @@ export enum CONFIG {
 	BIND_HOST = 'BIND_HOST',
 	PORT = 'PORT',
 	WS_URL = 'WS_URL',
+	HTTP_URL = 'HTTP_URL',
```

Comment:
This works! Another approach would be to just accept a URL and check whether it was a WS or HTTP one, rather than having a separate env var for each; that way, there's no need to wonder which is the default or handle both being provided or whatever :)

Reply:
I really like that idea :)

Reply:
Yea at first I wanted to do the same thing, but I didn't do it to avoid a breaking change. But that being said I think this holds merit and is a lot better in the long run! I'll switch it over